### DISCUSSION TITLE
Next.js Local Host shows a blank page for Index.js

### DISCUSSION THREAD

**Discussion by MacCode7110** [vercel/next.js#26819] [Help]
### What version of Next.js are you using?

10.0.1

### What version of Node.js are you using?

16.2.0

### What browser are you using?

Microsoft Edge

### What operating system are you using?

Windows 10

### How are you deploying your application?

Vercel

### Describe the Bug

I am new to website development and I am learning how to create a Next.js app. When I load my local host site on my browser, it is completely blank. My homepage is based on my index.js file. I don't think that the local host should be blank since I have reviewed and revised my code, but I am not 100% sure. 
Here is a link to my Github page:  https://github.com/MacCode7110/Personal-Music-Reviews-Website 

Any suggestions on what is causing the blank page and how to resolve it would be appreciated!

### Expected Behavior

I expect a list of links with images to album reviews to be displayed on the homepage. Since I have not added any CSS styling,   the links and images should not be neatly organized.

### To Reproduce

Here is the code for my index.js file:

import Head from 'next/head'
import Image from 'next/image'
import {Fragment} from 'react'; //React fragment blocks allows us to group togther h1 blocks, list blocks, and more for the purposes of returning from a functionmport styles from '../styles/Home.module.css'
import Link from 'next/link';
import {getMetalReviews} from '../reviewdata';
import reviewList from '../components/reviews/review-list'

function HomePage()
{
    const metalReviewList = getMetalReviews();

    return ( 
        <div>
             <reviewList reviews = {metalReviewList}/>
        </div>
            );
}

export default HomePage;

**Comment by rsipakov** [comment]
Try to use a PascalCase for React components. I'm not sure if it helps, but it caught my eye. Also, I think is better change this `<img> src = {'/' + image} alt = {title} </img>` to self closing tag `<img src = {'/' + image} alt = {title} />`


https://www.quora.com/Why-do-React-components-use-PascalCase-for-filenames

**Reply by MacCode7110** [reply]
I tried using a self closing tag instead, it doesn't seem to make the local host site display the images and links in any way,

**Comment by MacCode7110** [comment]
Any other suggestions about what could be breaking the page? I am really struggling with this issue.

**Reply by Osomhe1** [reply]
Having same issue here, what can we do to resolve it.

**Reply by magicspon** [reply]
I often see this in Brave.  Usually happens after killing the process then rerunning `next dev`.
If I fully close the browser and reopen, it works.

**Comment by edejohn** [comment]
I ran into the same issue. I later discovered that I was running "npm run dev" in the wrong directory.
Instead of "cd C:\wamp\www\projectname", I typed "cd c:\wamp\www\projectname".
The difference is capital letter "C". Small letter "c" won't work.
Ensure your casing (capital or small) is right and everything spelt correctly. Then, run "npm run dev" and see your expected page display (if there are no other errors hindering it from showing).
